### PR TITLE
Chore db seeds and migrations

### PR DESCRIPTION
**Pull request by DanEvans485**

### CODE REVIEW

File: database/seeds/WeaponSeeder.php [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+            'subtype_id' => $simple_ranged_item_subtype->id,
+        ]);
+
+        Item::create([
+            'name'       => 'Toaster',
+            'cost'       => 99999,
+            'weight'     => 1,
+            'type_id'    => $weapon_item_type->id,
+            'subtype_id' => $simple_ranged_item_subtype->id,
+        ]);
```
Comment (G-Bro) [inline database/seeds/WeaponSeeder.php]: 👀

File: .idea/OpenDM-Server.iml [hunk 2]
```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (G-Bro) [inline .idea/OpenDM-Server.iml]: oi oi, wots all this then?

Comment (DanEvans485) [reply .idea/OpenDM-Server.iml]: I started using phpStorm and it started adding shit to an .idea folder

File: database/seeds/WeaponSeeder.php (hunk 1 above)
Comment (DanEvans485) [reply database/seeds/WeaponSeeder.php]: Doesn't look like anything to me

File: app/Http/Controllers/API/APIController.php [hunk 3]
```diff
@@ -29,8 +33,38 @@ public function list(ApiListRequest $request)
...
     public function retrieve(int $id)
     {
-        $resource_class = "App\Http\Resources\\{$this->model_class}Resource";
+        dd(
+            Character::all(),
+            Monster::all(),
+            Player::all()
+        );
+
+        dd('done');
```
Comment (G-Bro) [inline app/Http/Controllers/API/APIController.php]: bonjour

File: app/Http/Controllers/API/APIController.php [hunk 4]
```diff
@@ -29,8 +33,38 @@ public function list(ApiListRequest $request)
...
+
+        $character->size()->associate(Size::findByCode('huge'));
+        $character->alignment()->associate(Alignment::findByCode('chaotic_evil'));
+        $character->hp_dice()->associate(Dice::find(4));
+
+        $character->save();
+
+        dd($character);
 
+        dd('done');
```
Comment (G-Bro) [inline app/Http/Controllers/API/APIController.php]: dd all the things

Comment (DanEvans485) [reply app/Http/Controllers/API/APIController.php]: ... I reverted dat >:l

**Pull request merged**